autoload/lsp/internal/document_hover/under_cursor.vim

```diff
@@ -101,18 +101,19 @@ function! s:show_preview_window(server_name, request, response) abort
         return
     endif
 
+    let l:lines = lsp#utils#_split_by_eol(join(l:contents, "\n\n"))
```

Comment:
What do you mean this "\n\n" ?

Reply:
The `l:contents` is meant the `multiple grouped contents`. So I think the `\n\n` is needed to separate content groups.

Reply:
Ah, make sense.